drivers/Serial.h

```diff
@@ -63,10 +63,10 @@ class Serial : public SerialBase, public Stream, private NonCopyable<Serial> {
      *  @param tx Transmit pin
      *  @param rx Receive pin
      *  @param name The name of the stream associated with this serial port (optional)
-     *  @param baud The baud rate of the serial port (optional, defaults to MBED_CONF_PLATFORM_DEFAULT_SERIAL_BAUD_RATE)
+     *  @param baud The baud rate of the serial port (optional, defaults to MBED_CONF_PLATFORM_DEFAULT_SERIAL_BAUD_RATE or 9600)
```

Comment:
@kjbracey-arm Yeah, `MBED_CONF_PLATFORM_DEFAULT_SERIAL_BAUD_RATE` is 9600

Reply:
@kjbracey-arm I wanted to add the `or 9600` because with just a quick glance I don't care about the `MBED_CONF_PLATFORM_DEFAULT_SERIAL_BAUD_RATE` configuration parameter, I want to know the actual numerical value it defaults to